src/kasn1/pyca/oid.rs

```diff
@@ -182,14 +182,16 @@ pub const SHA384_OID: asn1::ObjectIdentifier =
     asn1::oid!(2, 16, 840, 1, 101, 3, 4, 2, 2);
 pub const SHA512_OID: asn1::ObjectIdentifier =
     asn1::oid!(2, 16, 840, 1, 101, 3, 4, 2, 3);
+// The SHA3 ones are wrong in pyca
+// https://github.com/pyca/cryptography/issues/13331
```

Comment:
This changes the pyca header in ways that will probably make it diverge from upstream.

In this case I'd rather drop these ids, and add custom ones in src/kasn1/oid.rs